src/GitVersionCore/LibGitExtensions.cs

```diff
@@ -113,7 +113,7 @@ public static Commit FindCommitBranchWasBranchedFrom([NotNull] this Branch branc
                         mergeBaseCommit = findMergeBase,
                         branch = otherBranch
                     };
-                }).Where(b => b != null).OrderByDescending(b => b.mergeBaseCommit.Committer.When).ToList();
+                }).Where(b => b.mergeBaseCommit != null).OrderByDescending(b => b.mergeBaseCommit.Committer.When).ToList();
```

Comment:
Scratch my comment. I read the entire file and it cannot be null since the select returns a new anonymous object